packages/x-components/src/x-modules/scroll/store/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './emitters';
+export * from './module';
+export * from './types';
+export { initScrollComponentState } from './utils';
```

Comment:
I know the content of this file is just this function, but, what's the reason behind only exporting it instead of whole file? If we add more utilities we would need to refactor this to be able to use them outside our library.

In case it was unintended, I´ll leave the suggestion here:

```suggestion
export * from './utils';
```